orc8r/cloud/helm/orc8r/templates/_helpers.tpl

```diff
@@ -15,7 +15,7 @@ limitations under the License.
 app.kubernetes.io/name: {{ .Chart.Name }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 app.kubernetes.io/managed-by: helm
-app.kubernetes.io/part-of: magma
+app.kubernetes.io/part-of: orc8r
```

Comment:
For the service registry service, we need to differentiate between gateway and orc8r components (e.g. ListAllServices should only return orc8r services). The common way to associate components from different helm charts is to use the `part-of` label.